.github/workflows/lint-and-test.yml

```diff
@@ -35,6 +35,12 @@ jobs:
           fi
       - name: Run chart-testing (lint)
         run: ct lint --debug --config ./.github/configs/ct-lint.yaml --lint-conf ./.github/configs/lintconf.yaml
+      
+      - name: Run docs-testing (helm-docs)
+        uses: buttahtoast/helm-release-action@v2.0.1
+        with:
+          charts: "${{ steps.list-changed.outputs.changed_charts }}"
```

Comment:
As far as I can tell, this will pick up and try to compare the results of`helm-docs` on all the charts, not just the ones that are helm-docs enabled (have `README.md.gotmpl`). That will break the build on a change to any chart that is not yet enabled for helm-docs. One way around it is to generate a separate list of changed charts with helm-docs support and run the check only on them. For example:
```bash
          changed=$(ct --config ./.github/configs/ct-lint.yaml list-changed)
          charts=$(echo "$changed" | tr '\n' ' ' | xargs)
          if [[ -n "$changed" ]]; then
            echo "::set-output name=changed::true"
            echo "::set-output name=changed_charts::$charts"
          fi
          changed_helm_docs=$(for chart in $changed ; do if test -f "$chart/README.md.gotmpl" ; then echo "$chart" ; done)
          charts_helm_docs=$(echo "$changed_helm_docs" | tr '\n' ' ' | xargs)
          if test -n "$changed_helm_docs" ; then
            echo "::set-output name=changed_helm_docs::true"
            echo "::set-output name=changed_helm_docs_charts::$charts_helm_docs"
          fi
```

Reply:
We could place the same note like it exists in the image-updaters README

Reply:
Add the remaining charts to a file .helmdocsignore?

Reply:
I added the .helmdocsignore. Are you okay with that, @vladlosev ?

Reply:
Yes, looks like it will work. The only downside is the check will not be automatically enabled for a chart if one adds the template there - one will also have to remember (or to know) to delete the directory from .helmdocsignore.

Reply:
Okay but in my eyes this sounds reasonable as such a contributor will be familiar with helm-docs.

Reply:
I'll trust your opinion here, then.